pkg/logging/json_logger.go

```diff
@@ -129,7 +130,15 @@ func (jsonLogger *JSONLogger) PostProcess() error {
 func viewLogsJSONFile(lvopts LogViewOptions, stdout, stderr io.Writer, stopChannel chan os.Signal) error {
 	logFilePath := jsonfile.Path(lvopts.DatastoreRootPath, lvopts.Namespace, lvopts.ContainerID)
 	if _, err := os.Stat(logFilePath); err != nil {
-		return fmt.Errorf("failed to stat JSON log file ")
+		// FIXME: this is a workaround for the actual issue, not a real solution
```

Comment:
Could you add a link to the github issue?

Reply:
Done